search/views.py

```diff
@@ -32,21 +31,35 @@ def search(request):
                 "result"
             ]
             return render(request, "search/result.html", {"z_results": results})
+    # generic zip code form post
+    elif request.method == "GET":
```

Comment:
Does it become more maintainable if specify get "what" in request.GET
e.g. elif request.method == "GET" and "zipcode" in request.GET:

Reply:
I don't think so because GET can have a variable amount of parameters depending on what fields were filled in, so ideally when a get request was received it would just pass in whatever arguments are not null to the layer API. This could become an issue if we have more than 1 GET submitting form in the search page but I think at that point we would have to split up functionality anyways.